examples/solids/elasticity.c

```diff
@@ -0,0 +1,714 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+//                        libCEED + PETSc Example: Elasticity
+//
+// This example demonstrates a simple usage of libCEED with PETSc to solve
+//   elasticity problems.
+//
+// The code uses higher level communication protocols in DMPlex.
+//
+// Build with:
+//
+//     make elasticity [PETSC_DIR=</path/to/petsc>] [CEED_DIR=</path/to/libceed>]
+//
+// Sample runs:
+//
+//     ./elasticity -mesh ./meshes/cylinder8_672e_4ss_us.exo -degree 2 -E 1 -nu 0.3 -problem linElas -forcing mms
+//     ./elasticity -mesh ./meshes/cylinder8_672e_4ss_us.exo -degree 2 -E 1 -nu 0.3 -bc_zero 999 -bc_clamp 998 -problem hyperSS -forcing none -ceed /cpu/self
+//     ./elasticity -mesh ./meshes/cylinder8_672e_4ss_us.exo -degree 2 -E 1 -nu 0.3 -bc_zero 999 -bc_clamp 998 -problem hyperFS -forcing none -ceed /gpu/occa
```

Comment:
I think these sample runs need to be updated, as it appears that the mesh file has been renamed, so these runs will crash

Reply:
see above discussion, this is the right name but the meshes are not included in our repo

Reply:
Ah, I lost that comment. So what should I run?

Reply:
Please @jeremylt or @ArashMehraban consider removing these lines now or update them to show where the meshes are, since they provide an instruction that won't run. There is no point in providing a wrong instruction.

Reply:
updated

Reply:
Thanks